.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# default owners for everything in the repository.
+*   @Microsoft/accessibility-insights-web-code-owners
```

Comment:
They updated the canonical casing of the microsoft github org to lowercase recently, we should probably match that